pinning.go

```diff
@@ -498,6 +498,14 @@ func (s *Server) handleDeletePin(e echo.Context, u *util.User) error {
 		return err
 	}
 
+	if content.AggregatedIn > 0 {
+		if s.CM.IsZoneConsolidating(content.AggregatedIn) || s.CM.IsZoneAggregating(content.AggregatedIn) {
+			return fmt.Errorf("unable to unpin content while zone is consolidating or aggregating (pin: %d, zone: %d)", content.ID, content.AggregatedIn)
+		}
+		// even if the zone is pinned (active), we can allow the content to be unpinned
```

Comment:
this is not true, if it is a child of a pinned aggregate, do not delete it, as it will damage the aggregate